api/nakadi-event-bus-api.yaml

```diff
@@ -1260,7 +1261,7 @@ definitions:
           - 'hash' will use the event field(s) defined in 'partitioning_key_fields' property of `EventType` as a source
             for a hash function that will caclulate the partition where the event will be put. All events with the same
             value in this field(s) will go to the same partition, and consequently be ordered.
-          - 'user_defined' will use 'metadata'.'partition' property of incoming event to know the partition where to put
+          - 'user_defined' will use `metadata.partition` property of incoming event to know the partition where to put
```

Comment:
Yes, should be for the above `partitioning_key_fields` also, have missed that. :)